fronts-client/src/actions/Collections.ts

```diff
@@ -238,10 +238,18 @@ function getCollections(
         getCollectionActions(collectionResponse, getState)
       );
 
+      const actionsToBatch = flatten([
+        ...actions,
+        ...missingActions,
+        ...missingCollectionActions,
+      ]);
+
+      const noChangesAction = collectionActions.fetchSuccess([]);
```

Comment:
Might be worth leaving a comment here to indicate why we're dispatching this. Other devs may consider it and remove, introducing a bug – and there's no test to catch it AFAICS (not necessarily saying add a test, it may be time consuming!)

Reply:
added a comment for now, will look into adding a test too